Remotion/Web/Development.WebTesting.TestSite/Startup.cs

```diff
@@ -4,11 +4,24 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Remotion.Development.UnitTesting;
+using Remotion.ServiceLocation;
+using Remotion.Web.ExecutionEngine;
 
 namespace Remotion.Web.Development.WebTesting.TestSite
 {
   public class Startup
   {
+    private ServiceLocatorScope _serviceLocatorScope;
+    public Startup ()
+    {
+      //I'm not sure if this is what we need... I think I need some more info on the penultimate list item of the task.
+      var wxeUrlSettings = WxeUrlSettings.Create("~/bin/UrlMapping.xml");
+      var serviceLocator = DefaultServiceLocator.Create();
+      serviceLocator.RegisterSingle(() => wxeUrlSettings);
+      _serviceLocatorScope = new ServiceLocatorScope(serviceLocator);
+    }
+
```

Comment:
I think we can create an implementation of `WxeUrlSettings` and register it in the IoC container with a smaller position and it should use that instead of the default

Reply:
```
      var wxeUrlSettings = WxeUrlSettings.Create("~/bin/UrlMapping.xml");
      var defaultServiceLocator= DefaultServiceLocator.Create();
      defaultServiceLocator.RegisterSingle(() => wxeUrlSettings);
ServiceLocator.SetLocatorProvider(() => defaultServiceLocator);
```

That's the preferred pattern for the web application. We don't have extensible WxeUrlSettings if I remember correctly. We would simply create a parameterized instance during startup and register this.